trieste/acquisition/multi_objective/partition.py

```diff
@@ -11,42 +11,53 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-""" This module contains functions and classes for Pareto based multi-objective optimization. """
+"""This module contains functions of different methods for
+partitioning the dominated/non-dominated region in multi-objective optimization problems."""
 from __future__ import annotations
 
 from dataclasses import dataclass
+from typing import Any
 
 import tensorflow as tf
-from typing_extensions import Final
 
-from ..types import TensorType
-from .misc import DEFAULTS
+from ...types import TensorType
+from ...utils.misc import DEFAULTS
+from .dominance import non_dominated
 
 
-def non_dominated(observations: TensorType) -> tuple[TensorType, TensorType]:
+def prepare_default_non_dominated_partition_bounds(
+    observations: TensorType, anti_reference: TensorType, reference: TensorType
```

Comment:
is it useful to have `anti_reference` as an input with no default? it seems that we're always going to use a numerically stable `-Inf`, e.g. `1e-10`. Or is there a use-case for the anti-reference? If not, maybe allow `None` and set default internally to make the ehvi code a bit more readable?

Reply:
this makes a lot of sense, have changed `anti_reference` as an optional arg with default `-1e10`.